packages/kolibri-common/components/Cards/AccessibleChannelCard.vue

```diff
@@ -20,14 +23,18 @@
     </template>
     <template #belowTitle>
       <div>
-        <KTextTruncator
-          :text="coachString('numberOfResources', { value: contentNode.num_coach_contents })"
```

Comment:
Here in the current implementation of this AccessibleChannelCard we are using this `num_coach_contents` to show the number of resources, but Im not that shure if thats completely accurate. I think the number we should be showing here is the `total_resource_count` of the channel. But havent  updated yet just in case the current value has any other motivation @marcellamaki.

Reply:
yes, I think it should be `total_resource_count` @AlexVelezLl

Reply:
Just pushed this change!

Reply:
thank you!